README.md

```diff
@@ -1,33 +1,45 @@
+![Tangle Treasury Logo](https://github.com/Deep-Sea-888/Operation-Agreement/blob/main/smr_community_treasury_white_large.png?raw=true)
 
+<h2 align="center">Shimmer Community Framework</h2>
 
+<p align="center">
+  <a href="https://discord.iota.org/" style="text-decoration:none;"><img src="https://img.shields.io/badge/Discord-9cf.svg?logo=discord" alt="Discord"></a>
+  <a href="https://wiki.iota.org/shimmer/learn/governance/shimmer-governance-intro" style="text-decoration:none;"><img src="https://img.shields.io/badge/Wikipedia-Governance-blue" </a>
+  <a href="https://tangletreasury.org/" style="text-decoration:none;"><img src=https://img.shields.io/badge/TangleTreasury-.org-blue"
+</p>
 
+<p align="center">
+  <a href="#about">About</a> ◈
+  <a href="#contributions">Contributing</a> ◈
+  <a href="#joining-the-discussion">Joining the discussion</a> 
+</p>
 
-<!-- LICENSE -->
-## License
+---
 
-Distributed under the Apache License. See `LICENSE` for more information.
+## About
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+This repository contains governance documents that were collectively created by the IOTA and Shimmer community, with the support of the IOTA foundation. The community voted to adopt these frameworks as the operating processes and procedures for conducting Shimmer governance and managing the Shimmer community treasury.
 
+  - [Exhibit B](https://github.com/Tangle-Community-Treasury-DAO/Operation-Agreement/blob/main/Exhibit%20B%20-%20V1.md): Shimmer Governance Framework
+  - [Exhibit C](https://github.com/Tangle-Community-Treasury-DAO/Operation-Agreement/blob/main/Exhibit%20C%20-%20V1.md): Shimmer Community Grant Committee Framework
```

Comment:
```suggestion
  - [Exhibit C](https://github.com/Tangle-Community-Treasury-DAO/Operation-Agreement/blob/main/Exhibit%20C%20-%20V1.md): Shimmer Community Grant Committee Specification
```

Reply:
I think we should call it specification, Thats how we did it most of the time WDYT? So there is a framework and a specification, maybe avoids confusion